<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>IncasaciyaForm</class>
  <widget class="QWidget" name="IncasaciyaForm">
    <property name="windowModality">
      <enum>Qt::WindowModal</enum>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>658</width>
        <height>433</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Диалог инкасации</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <property name="margin">
        <number>0</number>
      </property>
      <item>
        <widget class="QWidget" name="IncasaciyaFormWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
              <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                  <number>12</number>
                </property>
                <item>
                  <widget class="QLabel" name="lblIncasaciyaTitle">
                    <property name="text">
                      <string>Инкасcация</string>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignCenter</set>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QTextBrowser" name="graphicsViewI" />
                </item>
                <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                      <widget class="QPushButton" name="btnChangePass">
                        <property name="autoFillBackground">
                          <bool>false</bool>
                        </property>
                        <property name="text">
                          <string>Сбросить
                            пароль</string>
                        </property>
                        <property name="iconSize">
                          <size>
                            <width>30</width>
                            <height>30</height>
                          </size>
                        </property>
                        <property name="autoDefault">
                          <bool>false</bool>
                        </property>
                        <property name="default">
                          <bool>false</bool>
                        </property>
                        <property name="flat">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnNullingCheck">
                        <property name="autoFillBackground">
                          <bool>false</bool>
                        </property>
                        <property name="text">
                          <string>Обнулить
                            счетчик чека</string>
                        </property>
                        <property name="iconSize">
                          <size>
                            <width>30</width>
                            <height>30</height>
                          </size>
                        </property>
                        <property name="autoDefault">
                          <bool>false</bool>
                        </property>
                        <property name="default">
                          <bool>false</bool>
                        </property>
                        <property name="flat">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnPrinterTest">
                        <property name="text">
                          <string>Тестовая
                            печать</string>
                        </property>
                        <property name="iconSize">
                          <size>
                            <width>30</width>
                            <height>30</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnInterToAdmin">
                        <property name="styleSheet">
                          <string notr="true" />
                        </property>
                        <property name="text">
                          <string>Админ
                            панель</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../../ekiosk.qrc">
                            <normaloff>:/assets/icons/reg_11.png</normaloff>
                            :/assets/icons/reg_11.png</iconset>
                        </property>
                        <property name="iconSize">
                          <size>
                            <width>48</width>
                            <height>48</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnGetIncasaciya">
                        <property name="styleSheet">
                          <string notr="true" />
                        </property>
                        <property name="text">
                          <string>Сделать
                            инкассацию</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../../ekiosk.qrc">
                            <normaloff>:/assets/icons/reg_10.png</normaloff>
                            :/assets/icons/reg_10.png</iconset>
                        </property>
                        <property name="iconSize">
                          <size>
                            <width>48</width>
                            <height>48</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <spacer name="horizontalSpacer">
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                          <size>
                            <width>40</width>
                            <height>20</height>
                          </size>
                        </property>
                      </spacer>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnCloseIncasaciya">
                        <property name="text">
                          <string>Закрыть
                            окно</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../../ekiosk.qrc">
                            <normaloff>:/assets/icons/reg_8.png</normaloff>:/assets/icons/reg_8.png</iconset>
                        </property>
                        <property name="iconSize">
                          <size>
                            <width>48</width>
                            <height>48</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources>
    <include location="../../ekiosk.qrc" />
  </resources>
  <connections />
</ui>